src/Kafka/CreateKafkaInstance/components/TrialKafkaForm.tsx

```diff
@@ -0,0 +1,198 @@
+  ToggleGroup,
+  ToggleGroupItem,
+  Tooltip,
+  Slider,
+  Text,
+  TextVariants,
+  ButtonVariant,
+} from "@patternfly/react-core";
+import { CloudRegionSelect } from "./CloudRegionsSelect";
+import { CloudProvidersTiles } from "./CloudProviderTiles";
+import { FormGroupWithPopover } from "../../../shared/FormGroupWithPopover";
+import { StandardKafkaFormProps } from "./StandardKafkaForm";
+
+export type TrialKafkaFormProps = Omit<
+  StandardKafkaFormProps,
+  "streamingUnits"
+>;
+
+export const TrialKafkaForm: VFC<TrialKafkaFormProps> = ({
+  FORM_ID,
+  isNameTaken,
+  isNameInvalid,
+  nameValidation,
+  name,
+  disableControls,
+  providerValidation,
+  availableProviders,
+  provider,
+  regionValidation,
+  regions,
+  region,
+  azValidation,
+  azOptions,
+  az,
+  disableAZTooltip,
+  isDisabledSize,
+  size,
+  setSize,
+  setRegion,
+  setAZ,
+  setProvider,
+  setName,
+  onSubmit,
+  isTesting,
+}) => {
+  const { t } = useTranslation("create-kafka-instance");
+
+  return (
+    <Form onSubmit={onSubmit} id={FORM_ID}>
+      <FormGroup
+        label={t("instance_name")}
+        helperText={t("create_instance_name_helper_text")}
+        helperTextInvalid={
+          isNameTaken
+            ? t("create_instance_name_helper_text_name_taken", { name })
+            : isNameInvalid
+            ? t("create_instance_name_helper_text")
+            : t("common:required")
+        }
+        isRequired
+        validated={nameValidation}
+        fieldId="form-instance-name"
+      >
+        <TextInput
+          id="form-instance-name"
+          isRequired
+          validated={nameValidation}
+          type="text"
+          value={name || ""}
+          onChange={setName}
+          autoFocus={true}
+          isDisabled={disableControls}
+        />
+      </FormGroup>
+      <FormGroup
+        label={t("cloud_provider")}
+        fieldId="form-cloud-provider-name"
+        validated={providerValidation}
+        helperTextInvalid={t("common:required")}
+        isRequired
+      >
+        <CloudProvidersTiles
+          providers={availableProviders}
+          value={provider}
+          onChange={setProvider}
+          isDisabled={disableControls}
+        />
+      </FormGroup>
+      <FormGroup
+        label={t("cloud_region")}
+        fieldId="form-cloud-region-option"
+        isRequired
+        validated={regionValidation}
+        helperTextInvalid={t("common:required")}
+      >
+        <CloudRegionSelect
+          value={region}
+          regions={regions || []}
+          onChange={setRegion}
+          isDisabled={disableControls}
+          validated={regionValidation}
+        />
+        {isTesting && (
+          <>
+            <Text component={TextVariants.p} className="pf-c-form__helper-text">
+              {t("cloud_region_description")}
+            </Text>
+            <Button variant={ButtonVariant.link} isInline>
+              {t("learn_about_cloud_regions")}
+            </Button>
```

Comment:
Ideally, the font size for this link would be the same size as the helper text. The only way to do that is by wrapping this button with the `className="pf-c-form__helper-text"` like you have above.